xcape/core.py

```diff
@@ -61,6 +63,10 @@ def _reshape_surface_outputs(*args, shape=None):
         target_shape = (1,) + shape[1:]
     return [np.reshape(a, target_shape) for a in args]
 
+def _reshape_outputs_stdheight(*args, shape=None):
+    # calc_cape needs input in shape (nlevs, npoints)
+    target_shape = shape
+    return [np.reshape(a, target_shape) for a in args]
```

Comment:
the output of stdheight is not just time,lat,lon but it has level dimension as well.